### ISSUE TITLE
Breadcrumbs lock the scroll on menu

### ISSUE THREAD

**Issue by eksperimental** [elixir-lang/ex_doc#254]
~~This one happens only in firefox for linux (latest version)~~
Found in Firefox for Linux

visit any page, let's say home page.
click on the last item in the breadcrumb, the top div, goes a bit to the left.
overlaps the scroll bar in the menu.
and locks that scrollbar, without being able to scroll down the menu.
WEIRD!

Update:
It does happen if you visit directly the page including "#content"
for example: http://elixir-lang.org/docs/master/elixir/Version.html#content

**Comment by josevalim** [comment]
Haha, since it is very specific, please try to come with a fix for this one if you can! Or open up a bug report on Firefox! :D

**Comment by eksperimental** [comment]
Let me rephrase myself.
I have only tested on linux, and I found this in Firefox.

**Comment by eksperimental** [comment]
It does not happen only when you click, but if you link directly to "#content",
for example: http://elixir-lang.org/docs/master/elixir/Version.html#content

**Comment by josevalim** [comment]
@eksperimental I may have fixed this in master. We were setting a negative margin on the right side of the breadcrumbs and that could have been the issue.

**Comment by eksperimental** [comment]
sorry @josevalim, but it did not fix it.
@milmazz is there anything that we are doing with javascript checking for "#content", because when i directly visit http://elixir-lang.org/docs/master/elixir/Version.html#content
and that may be the hint that there is a js function doing something to that div.
thank you.

**Comment by milmazz** [comment]
@eksperimental Appears to be a problem related w/Firefox, I made a few tests with Chrome/Safari and everything appears to be working smoothly. But when I visit a page (w/Firefox) with any anchor link the `#content` section looks `-10px` to the left (but the computed CSS rules and the box model appears to be the same in both scenarios -with and without anchor link-, which is weird). I don't know why is this behaviour in Firefox, I can dig into this later.

@Dignifiedquire What do you think?

**Comment by Dignifiedquire** [comment]
@milmazz fixed, see attached PR

**Comment by eksperimental** [comment]
I am reopening this as #274 didn't fixit.

I have the feeling it is related to this line of code
https://github.com/elixir-lang/ex_doc/blob/master/assets/js/events.js#L344

and the offcanvas-navigation thing.
I tried replacing the links to #content  with #sidebar and it's not giving that issue (at least in firebug)

I'm trying to install but I cannot get it working, so i'm just reporting it here for now.

/cc @milmazz @Dignifiedquire

**Comment by milmazz** [comment]
@eksperimental Is this issue still valid? Also apply for the `new_design` branch?

**Comment by eksperimental** [comment]
@milmazz I haven't tested out new_design, once you merged, I will.
please ping me if I forget

**Comment by milmazz** [comment]
@eksperimental `new_design` is already merged into `master`

**Comment by Dignifiedquire** [comment]
Breadcrumbs are no more, so I think it's safe to close this. Please reopen @eksperimental if you still find this issue.
